app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.test.tsx

```diff
@@ -89,19 +89,16 @@ describe("AppLayoutWithExperiment", () => {
         mocks={[mock]}
         redirect={({ status }) => {
           if (status.review) {
-            return "request-review";
+            return "";
```

Comment:
I don't think it matters, the user will be taken to `/nimbus/${slug}` vs `/nimbus/${slug}/`

Reply:
Just catching up on this, we should be careful about this in general, Django gets cranky for urls that are missing a trailing slash.  In this case it's handled by the react router, so Django probably won't care, but it could lead to wonkiness in the future so we should generally try to make sure we include trailing slashes.